src/lib/tabs/_tabs-common.scss

```diff
@@ -24,7 +24,10 @@ $mat-tab-animation-duration: 500ms !default;
 
   &.mat-tab-disabled {
     cursor: default;
-    pointer-events: none;
+
+    &:focus {
```

Comment:
Fading out the element only on focus seems a little weird. Wouldn't it be better if we prevented focus in the first place? That's what the `pointer-events` approach was doing.

Reply:
By disabling `pointer-events` entirely things like tooltips are prevented from being attached to a `<mat-tab-label>`